webapp/src/Controller/PublicController.php

```diff
@@ -100,6 +100,14 @@ public function scoreboardDataZipAction(
         return $this->dj->getScoreboardZip($request, $requestStack, $contest, $this->scoreboardService);
     }
 
+    #[Route(path: '/dynamic-css', name: 'get_dynamic_css')]
+    public function dynamicCSS(): Response {
+        $response = new Response();
+        $response->headers->set('Content-Type', 'text/css');
```

Comment:
We should maybe set some expire headers? Otherwise browsers won't cache it.
Drawback is that it will change once the colors change, so maybe use an etag header instead?

Reply:
We discussed and I'll try to implement a cache and keep the etag there.